vital/types/mesh.cxx

```diff
@@ -419,6 +419,15 @@ ::merge(const mesh& other)
     {
       tex = std::vector<vector_2d>(2*num_e, vector_2d(0,0));
     }
+    else if (other.has_tex_coords() == TEX_COORD_ON_FACE_CORNER)
+    {
+      int nb_face_corners = 0;
+      for (int f=0; f < this->num_faces(); ++f)
```

Comment:
Very minor nit. Put spaces between operators and operands. Same comment applies for all cases below.